test/RootStateSchema.json

```diff
@@ -312,9 +350,7 @@
                     "type": "string"
                 },
                 "category": {
-                    "enum": [
```

Comment:
change made from this PR https://github.com/YousefED/typescript-json-schema/pull/535